dense_vector/track.py

```diff
@@ -205,6 +217,12 @@ async def __call__(self, es, params):
             knn_hits = [hit["_id"] for hit in knn_result["hits"]["hits"]]
             script_hits = await knn_vector_store.get_neighbors_for_query(index, query_id, target_k, request_cache, es)
             script_hits = script_hits[:k]
+            vector_operations_count = 0
+            profile = knn_result["profile"]
+            for shard in profile["shards"]:
```

Comment:
The number of nodes visited depends heavily on number of shards and segments, which is something that we need to take into account when interpreting the results (i.e. removing the `force_merge` action into a single segment will sky-rocket this value).